pyOCD/transport/cmsis_dap.py

```diff
@@ -381,6 +381,8 @@ def _write(self, request, data = 0):
         """
         Write a single command
         """
+        assert type(request) is int, "request is not an int"
+        assert type(data) is int, "data is not an int"
```

Comment:
Ah, it's dependant on whether your machine is 32- or 64-bit. I'm changing it to `in (int, long)`.